### ISSUE TITLE
Why not merge into Libertinus

### ISSUE THREAD

**Issue by alerque** [StefanPeev/Common-Serif#62]
Can I ask what the rational is for not developing this as an actual fork that could be contributed back to the Libertinus project? There has been quite a bit of discussion and even I've expressed interest in redoing the font based on masters that could be interpolated but I don't recall ever seeing somebody that was ready to jump in and try. What is the current status of development, how much has the design diverged from Libertinus Serif at comparable instance weights? Are there significant design changes that would make it *not* a good candidate for eventually replacing the Libertinus Serif static instances?

**Comment by StefanPeev** [comment]
@alerque I'm afraid that combining the two projects is no longer possible. The genetic kinship of the two typefaces is not sufficient to join Common Serif to Libertinus, as each grapheme in Common Serif is already constructed from a different number of contour points compared to Libertinus Serif, and there are generally significant differences in contour construction.
The mechanical unification of the two projects will not give a positive result, since the projects lie on a different principle basis. In Libertinus Serif, each style is developed for itself, while in Common Serif, the tactics of the masters styles is applied, which define the possibility of automatically generated intermediate instances.

**Comment by StefanPeev** [comment]
@alerque The overall design is largely the same and metrics are close to compatible with Libertinus Serif. Besides, Cyrillic part of the font family is extended and now it includes localisation for the Bulgarian language. The Greek part is also extended significantly and Polytonic Greek combining glyphs like Greek Capital Letter Alpha With Psili are generated from the basic glyph plus the diacritics (for example: Alpha plus Psilli (uni1FBD)), which make the glyphs' design more compatible.

![Screenshot 2024-09-27 160816](https://github.com/user-attachments/assets/fceefd9b-5bf0-4e7c-9122-3f6d79007b95)

**Comment by alerque** [comment]
Libertinus Serif isn't even internally consistent with metrics and weights. The 400 weight Regular is pretty much the base point and the next-best consistency wise is Regular Bold, but the italic variants vary a bit inconsistently. In between those the semibold weight is just wildly inconsistent and doesn't even have the same glyph coverage.

I would also not expect a VF based overhaul to be 100% metrically compatible, but it would greatly ease things if the existing main instances were close enough to something on the VF axis that it could be explored as a replacement.

I'd like to further explore integrating this back into the project. I would imagine the steps being (roughly) something like:

1. Introduce a new family, *Libertinus Serif VF* — at least initially not touching the current static instances.
2. Bring in the masters and work out the build system such that the VF can be built and released (and compared) side by side with the current static fonts.
3. Release with both the current static fonts and a new VF face to get some broad usage and testing.
5. Generate static instances from the VF, potentially under a temporary name so they can be used in parallel during a trial phase.
4. Make sure the glyph coverage of the VF edition is fully up to par with the current static instances.
5. Make sure a few basic points in the new VF's design space (400/Regular, 400/Italic, 700/Bold, 700/BoldItalic) corresponds fairly closely to the metrics of the current instances. Obviously there would be some differences (hopefully all for the better) but I would expect *most* current documents to give the same basic metrics (baseline skip, overall grade) even if there were a few kerning or contour differences here and there.
6. Ditch the current static instances entirely and replace them with instances generated from the VF for improved consistency.
7. Release the all-new static instances and matching VF as a new major version bump.

Is this something this project is interested in seeing happen? Or is taking off in a different direction and not expecting to bring existing Libertinus users along for the ride preferred? (Understanding that the latter choice may mean we eventually get a VF overhaul for Libertinus which largely overlaps with this project.)

**Comment by StefanPeev** [comment]
@alerque If you offer a working VF font project, you already offer a family of static versions of the font. You can use Common Serif as a base to create Libertinus Serif LGC Pro - as a variant of the currently existing static versions of Libertinus Serif. So the project can continue to develop both as a VF font version and as new static varieties of Libertinus Serif under the new name (it is a good idea the name to be shorter then Libertinus Serif LGC Pro).
In this regard, it is good to seek the opinion of @davelab6 (Dave Crossland) as well, as he has made a proposal [#7981](https://github.com/google/fonts/issues/7981) for the parallel inclusion of Common Serif and Libertinus Serif to the Google Fonts projects.
